resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -89,9 +95,9 @@ private[spark] object Config extends Logging {
 
   val KUBERNETES_DRIVER_SUBMIT_CHECK =
     ConfigBuilder("spark.kubernetes.submitInDriver")
-    .internal()
-    .booleanConf
-    .createWithDefault(false)
+      .internal()
```

Comment:
this is a nit, but pure formatting changes are not good PR practice.

Reply:
fixed it. ide formated.